packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -282,7 +283,11 @@ const AdvertiserPage = ({ navigate, selected_advert, showVerification }) => {
                         <Table.Cell className='advertiser-page__stats-cell'>
                             <div className='advertiser-page__stats-cell-header'>{localize('Avg. release')}</div>
                             <div className='advertiser-page__stats-cell-info'>
-                                {release_time_avg ? localize('{{release_time_avg}} min', { release_time_avg }) : '-'}
+                                {release_time_avg
+                                    ? localize('{{- avg_release_time}} min', {
+                                          avg_release_time,
+                                      })
```

Comment:
- This won't work, the `-` should be outside the placeholder.
- We could give the translators some extra context by renaming the placeholder to something like `avg_release_time_in_minutes`.

```suggestion
                                    ? localize('- {{ avg_release_time }} min', { avg_release_time })
```

Reply:
- This is the only way I can show `< 1`, otherwise in every other way I tried, `< ` becomes `&lt;`. I've tried using `&lt;` as well and even then it'd show just that.

- I like `avg_release_time_in_minutes`, sounds good. 👍

Reply:
You'll have to use two `localize` calls in that case, because our script won't find the placeholder (i.e. it sees `avg_release_time` but it doesn't see `- avg_release-time`.